services/QuillLMS/client/app/bundles/Teacher/components/classrooms/import_google_classroom_students_modal.tsx

```diff
@@ -30,12 +30,11 @@ export default class ImportGoogleClassroomStudentsModal extends React.Component<
     this.handleImportStudents = this.handleImportStudents.bind(this)
   }
 
-  initializePusherForGoogleStudentImport(id) {
-    if (process.env.RAILS_ENV === 'development') {
-      Pusher.logToConsole = true;
-    }
+  initializePusherForGoogleStudentImport(userId) {
+    if (process.env.RAILS_ENV === 'development') { Pusher.logToConsole = true; }
```

Comment:
Not worth changing in this PR since you're just changing formatting here, but I wonder if this is something that we can configure at a higher level.

Reply:
I'll make a note to take a look when I tackle the frontend work.